app/assets/stylesheets/helpers/_header.scss

```
@@ -446,19 +447,24 @@
   }
 
   .survey-form-button {
-    @include button(govuk-colour("white"));
     @include bold-19($line-height: 1.1);
+    padding: 10px 15px 5px 15px;
+    margin-right: .5em;
     color: $govuk-brand-colour;
+    background: white;
+    border: 0;
+    border-bottom: solid 2px silver;
 
     &:hover,
     &:focus {
       color: darken($govuk-brand-colour, 5%);
+      background: #f2f2f2;
```

Could this be replaced with `govuk-colour("light-grey")` which will compile to `#f3f2f1`?
```suggestion
      background: govuk-colour("light-grey");
```